### DISCUSSION TITLE
Embedded Mongo testing installs wrong linux platform

### DISCUSSION THREAD

**Discussion by sc6l6d3v** [Kirill5k/mongo4cats#48] [Q&A]
When I run this test in your repo:

src/test/scala/mongo4cats/collection/MongoCollectionAggregateSpec.scala

I get this .embedded directory:

(cd ~/.embedmongo; find . -ls)
 41205781      4 drwxrwxr-x   4  hkatz    hkatz        4096 Aug 14 22:01 .
 41205782      4 drwxrwxr-x   3  hkatz    hkatz        4096 Aug 14 22:01 ./archives
 41205785      4 drwxrwxr-x   3  hkatz    hkatz        4096 Aug 14 22:01 ./archives/https-fastdl-mongodb-org
 41205786      4 drwxrwxr-x   3  hkatz    hkatz        4096 Aug 14 22:01 ./archives/https-fastdl-mongodb-org/56cb3d10404e5b9df5358ac4ee83efcb
 41205787      4 drwxrwxr-x   3  hkatz    hkatz        4096 Aug 14 22:01 ./archives/https-fastdl-mongodb-org/56cb3d10404e5b9df5358ac4ee83efcb/linuxmongodb-linux-x86-64-rhel80-5-0-14-tgz
 41205788      4 drwxrwxr-x   2  hkatz    hkatz        4096 Aug 14 22:01 ./archives/https-fastdl-mongodb-org/56cb3d10404e5b9df5358ac4ee83efcb/linuxmongodb-linux-x86-64-rhel80-5-0-14-tgz/f1e5f76664247e125f251668a97eefed95e9bd617290f56d346c58ddb25e391b
 41169227  84556 -rw-------   1  hkatz    hkatz    86581082 Aug 14 22:01 ./archives/https-fastdl-mongodb-org/56cb3d10404e5b9df5358ac4ee83efcb/linuxmongodb-linux-x86-64-rhel80-5-0-14-tgz/f1e5f76664247e125f251668a97eefed95e9bd617290f56d346c58ddb25e391b/archive.tgz
 41205783      4 drwxrwxr-x   4  hkatz    hkatz        4096 Aug 14 22:01 ./fileSets
 41205789      4 drwxrwxr-x   2  hkatz    hkatz        4096 Aug 14 22:01 ./fileSets/c86618da1c63fc55dd6c13b0c77aa76b00dc74f5f3138937c222a75a631f23cd
 41173318 107540 -rwxrw-r--   1  hkatz    hkatz    110116168 Aug 14 22:01 ./fileSets/c86618da1c63fc55dd6c13b0c77aa76b00dc74f5f3138937c222a75a631f23cd/mongod
 41205784      4 drwxrwxr-x   2  hkatz    hkatz         4096 Aug 14 22:01 ./fileSets/hashes
 41169303      4 -rw-rw-r--   1  hkatz    hkatz           64 Aug 14 22:01 ./fileSets/hashes/0ea962fedf47924e59db7e2ff30ebb38a86dac5d2afb3090ecd0e7779bbe0647

But when I copy the exact same test into my test directory in my repo (https://github.com/sc6l6d3v/ratingslave), which uses mongo4cats, http4s and CE 3.x and attempt to run it, the failure is as a result of downloading ubuntu:

(cd ~/.embedmongo; find . -ls)
 41205790      4 drwxrwxr-x   4  hkatz    hkatz        4096 Aug 14 23:27 .
 41205791      4 drwxrwxr-x   3  hkatz    hkatz        4096 Aug 14 23:27 ./archives
 41205796      4 drwxrwxr-x   3  hkatz    hkatz        4096 Aug 14 23:27 ./archives/https-fastdl-mongodb-org
 41205797      4 drwxrwxr-x   3  hkatz    hkatz        4096 Aug 14 23:27 ./archives/https-fastdl-mongodb-org/56cb3d10404e5b9df5358ac4ee83efcb
 41205798      4 drwxrwxr-x   3  hkatz    hkatz        4096 Aug 14 23:27 ./archives/https-fastdl-mongodb-org/56cb3d10404e5b9df5358ac4ee83efcb/linuxmongodb-linux-x86-64-ubuntu2004-5-0-14-tgz
 41205799      4 drwxrwxr-x   2  hkatz    hkatz        4096 Aug 14 23:27 ./archives/https-fastdl-mongodb-org/56cb3d10404e5b9df5358ac4ee83efcb/linuxmongodb-linux-x86-64-ubuntu2004-5-0-14-tgz/62fa5e5dc65482463afa9081cf2bd981e67390f36b0f3d50744e200f78ce3884
 41193384  84552 -rw-------   1  hkatz    hkatz    86576813 Aug 14 23:27 ./archives/https-fastdl-mongodb-org/56cb3d10404e5b9df5358ac4ee83efcb/linuxmongodb-linux-x86-64-ubuntu2004-5-0-14-tgz/62fa5e5dc65482463afa9081cf2bd981e67390f36b0f3d50744e200f78ce3884/archive.tgz
 41205794      4 drwxrwxr-x   4  hkatz    hkatz        4096 Aug 14 23:27 ./fileSets
 41205800      4 drwxrwxr-x   2  hkatz    hkatz        4096 Aug 14 23:27 ./fileSets/f37d2ef0a53cd5272a04d0dff6e8421a0aa9acc8acce8d385a06290151dbfe1f
 41196162 107540 -rwxrw-r--   1  hkatz    hkatz    110113392 Aug 14 23:27 ./fileSets/f37d2ef0a53cd5272a04d0dff6e8421a0aa9acc8acce8d385a06290151dbfe1f/mongod
 41205795      4 drwxrwxr-x   2  hkatz    hkatz         4096 Aug 14 23:27 ./fileSets/hashes
 41195700      4 -rw-rw-r--   1  hkatz    hkatz           64 Aug 14 23:27 ./fileSets/hashes/2ecc4d77a60e7b012a83165981ef376c5eab19b223a05058dc07f816572ceb80

I am on Alma Linux 8.10, build is using JDK-21. Any suggestions how to fix the wrong selection of this de.flapdoodle library?

Thanks!

**Comment by sc6l6d3v** [comment]
Resolution was updating to the latest version of your repo which imported the version of de.flapdoodle which recognized alma-linux.

**Reply by Kirill5k** [reply]
Sorry, I didn't respond on time. Glad you managed to sort it out 👍
